tests/test_moviepy_setup.py

```
@@ -0,0 +1,40 @@
+# test_moviepy_setup.py
+
+import pytest
```

Ruff

`pytest` imported but unused

Remove unused import: `pytest`

(F401)